core/drivers/clk/sam/at91_pmc.h

```diff
@@ -137,7 +137,36 @@
 #define     AT91_PMC_PLLADIV2_ON		BIT(12)
 #define	  AT91_PMC_H32MXDIV	BIT(24)
 
+/* definitions for the PMC register of SAMA7G5 */
+#define AT91_PMC_MCR_V2			0x30
+#define AT91_PMC_MCR_V2_ID_MSK		0x0F
+#define AT91_PMC_MCR_V2_ID(_id)		((_id) & AT91_PMC_MCR_V2_ID_MSK)
+#define AT91_PMC_MCR_V2_CMD		SHIFT_U32(1, 7)
+#define AT91_PMC_MCR_V2_DIV_MSK		SHIFT_U32(7, 8)
+#define AT91_PMC_MCR_V2_DIV1		SHIFT_U32(0, 8)
+#define AT91_PMC_MCR_V2_DIV2		SHIFT_U32(1, 8)
+#define AT91_PMC_MCR_V2_DIV4		SHIFT_U32(2, 8)
+#define AT91_PMC_MCR_V2_DIV8		SHIFT_U32(3, 8)
+#define AT91_PMC_MCR_V2_DIV16		SHIFT_U32(4, 8)
+#define AT91_PMC_MCR_V2_DIV32		SHIFT_U32(5, 8)
+#define AT91_PMC_MCR_V2_DIV64		SHIFT_U32(6, 8)
+#define AT91_PMC_MCR_V2_DIV3		SHIFT_U32(7, 8)
+#define AT91_PMC_MCR_V2_CSS_SHIFT	16
+#define AT91_PMC_MCR_V2_CSS_MSK		SHIFT_U32(0x1F, 16)
```

Comment:
prefer `GENMASK_32(20, 16)` for defining a mask value.
This comment applies to `AT91_PMC_MCR_V2_DIV_MSK` and `AT91_PMC_MCR_V2_CSS_MSK`.

By the way, i would suggest to use explicit `_MASK` suffix instead of `_MSK` as it cost 1 more character to make it clear the macro refers to a mask.